### ISSUE TITLE
Victim/nemesis for each player

### ISSUE THREAD

**Issue by ChrisNeedham24** [jamesgiu/quick-hit#8]
Victim = The player the selected player beats the most
Nemesis = The player the selected player loses to the most

This could be either as a raw score or as a percentage of games played (if this was used, there would have to be a minimum games played threshold).

**Comment by ChrisNeedham24** [comment]
Would it be worth doing a first cut of this client side?

**Comment by jamesgiu** [comment]
Hmmm up to you, I think there's a cool opportunity to add a Badge object to the schema for the other issue and we can reuse that for victim/nemesis but I'm cool with either approach

**Comment by ChrisNeedham24** [comment]
Well I was thinking because the player stats page already goes through all of the player’s matches to get the W-L record, that for loop could be combined with the min/max ELO one and then this as well to get all statistics at once. That way it wouldn’t be any less efficient. This would only be a temporary thing anyway, though, eventually we’d make it DB-based.

**Comment by jamesgiu** [comment]
Yeah that makes sense - I'm cool with it if you were feeling inspired to do it like that. For the profile pages it'll be the same like you said,  however displaying the info on the ladder page clientside will be more complex to keep the iterations down

**Comment by ChrisNeedham24** [comment]
I wasn’t actually thinking of putting the info on the ladder page. What did you have in mind for that?

**Comment by jamesgiu** [comment]
New columns or icons next to the names that signify who is currently a nemesis of who I reckon, but that can be a future thing

**Comment by ChrisNeedham24** [comment]
This issue is more so for the backend implementation of this now, as the client-side implementation has been done.
